pkg/services/ngalert/metrics/ngalert.go

```diff
@@ -179,15 +179,15 @@ func newSchedulerMetrics(r prometheus.Registerer) *Scheduler {
 				Namespace: Namespace,
 				Subsystem: Subsystem,
 				Name:      "schedule_alert_rules",
-				Help:      "The number of alert rules being considered for evaluation each tick.",
+				Help:      "The number of alert rules that could be considered for evaluation at the next tick.",
```

Comment:
I'm not sure about **at the next tick** as it also represents the alert rules being considered for evaluation at the current tick and at previous ticks if there were no changes?